app/lib/controllers/api-bridge.js

```diff
@@ -57,21 +57,35 @@ APIBridgeController.prototype.post = function (req, res, next) {
         clientId: apiConfig.credentials.clientId,
         secret: apiConfig.credentials.secret
       }).then(request => {
-        return request({
+        let payload = {
           uri: requestObject.uri.href,
           method: requestObject.method,
-          body: typeof requestObject.body !== 'undefined' ? requestObject.body : null
-        }).then(response => JSON.parse(response))
+          headers: {
+            'content-type': 'application/json'
+          }
+        }
+
+        if (typeof requestObject.body !== 'undefined') {
+          payload.body = JSON.stringify(requestObject.body)
+        }
+
+        return request(payload).then(response => JSON.parse(response))
       }))
     })
 
     Promise.all(queue).then(response => {
-      let output = multiple ? response : response[0]
+      const output = multiple ? response : response[0]
```

Comment:
@eduardoboucas as discussed, `const` inside a method is not supported pre node 6.4 and 4.2 LTS end of support is 1st April 2018